### ISSUE TITLE
Make maintained theme properties default; deprecate old properties

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#10626]
In Firefox 69 the following deprecated theme properties will no longer work, so the alternate properties should be used instead for any generated (wizard) static themes.  **All the maintained properties work already in the preview** but are generally treated as alternate, or Chrome-only, variants, so the logic should ideally change.

Maintained | Deprecated | Deprecated used in wizard
-- | -- | --
theme_frame | headerURL | yes
frame | accentcolor | yes
tab_background_text | textcolor | yes
bookmark_text | toolbar_text | yes

**Comment by eviljeff** [comment]
We probably want to _remove_ support entirely for the deprecated theme properties but that's not a requirement.  Though if we don't we should arguably add detection and set the max-version appropriately.

**Comment by eviljeff** [comment]
QA: the following scenarios should work -
* uploading a pre-packaged static theme using the new properties
* using the wizard for a new theme
* using the wizard for a new version of an older theme (i.e. one that used the old properties) - the old properties and background image should be retained still

Uploading a version using the old properties isn't supported so I'm not sure the outcome will be - I _think_ the colors will still work.

**Comment by AlexandraMoga** [comment]
Verified on AMO -dev with FF66, WIn10x64

Here are my test results:
- uploading a new static theme through the wizard - I've downloaded the generated package and inspected the manifest.json file to make sure that the new theme properties are applied. There is one issue here with `toolbar-field` for which I filed #11004 
- uploading a theme `.xpi` using the new properties - the theme preview is rendered after submission according to the specified properties
- uploading a new version through the wizard for a theme using the old properties (previous version submitted either through the wizard, or pre-packed) - the old background and color properties are preserved 
- there are validation warnings triggered for packed themes using deprecated properties, together with a link that redirects the user to the new properties that should be used


@eviljeff regarding your last comment, I see that currently the linter is throwing only a warning for themes using the old properties, so it is still possible to upload these type of themes. The colors are still displayed accordingly in the preview generated by AMO and in the browser as well.
